base/uk.ac.stfc.isis.ibex.ui.about/src/uk/ac/stfc/isis/ibex/ui/about/AboutDialogBox.java

```diff
@@ -52,13 +61,23 @@ public AboutDialogBox(Shell parentShell, String applicationName) {
 		super(parentShell);	
 		setShellStyle(getShellStyle() | SWT.RESIZE | SWT.MAX);
 		this.applicationName = applicationName;
+		
+	      try {
+			image = new Image(null, new FileInputStream("C:\\Instrument\\Dev\\ibex_gui\\base\\uk.ac.stfc.isis.ibex.e4.product\\icons\\IBEX-icon-web48.gif"));
+		} catch (FileNotFoundException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
```

Comment:
Probably shouldn't be using print on catching an exception, if we want to alert people its probably better to use a logger? that seems to be the standard for Ibex.

Also it would be nice to remove the `// TODO comment`